### ISSUE TITLE
Unit

### ISSUE THREAD

**Issue by Nikita-Boyarskikh** [KurakovEgor/Project_One#5]
* [ ] Кажется, правильнее перенести сюда `speed`, `direction`, accelerate`, `controlByUser`
* [ ] Конструктор сделать с параметрами по умолчанию + в параметрах объект базового класса (в остальных классах-наследниках также) чтобы можно было собрать сначала `Image`, потом из него `Sprite`, потом из него `Unit`, потом из него уже кого-то конкретного

**Comment by KurakovEgor** [comment]
не согласен, так как неподвижные tower'ы не имеют скорости и направления движения, насчёт возможности controlByUser неподвижными объектами стоит подумать, по второму замечанию согласен, в TODO
